### ISSUE TITLE
Question about accuracy function in ImageNet example

### ISSUE THREAD

**Issue by zhangdan8962** [pytorch/examples#873]
Hi,

For accuracy in DistributedDataParallel setting, I am wondering how accuracy is gathered from all gpus? I am so confused because I think the accuracy is only calculated from the data on each gpu, which means it is not for the whole dataset.

Thank you in advance for the help!

**Comment by PoonKinWang** [comment]
The Distributed Data Parallel (DDP) wraps the model used to [output the gathered predictions from all devices](https://github.com/pytorch/pytorch/blob/master/torch/nn/parallel/scatter_gather.py). 
The number of GPUs may restricts the experiment to achieve the accuracy provided by original paper. Distributed-BatchNorm-PyTorch(DBP) can simulates multi-GPUs to train the network.
